setu/deeplink.py

```diff
@@ -62,12 +63,16 @@ def create_payment_link(
 
         # Generate required headers
         headers = generate_setu_headers(
-            self.schemeId, self.secret, self.productInstance
+            self.schemeId, self.secret, self.productInstance, self.url, self.authType
```

Comment:
Please add changes to the status check and credit alert APIs too

Reply:
Done.